server/public/shared/request/context.go

```diff
@@ -46,6 +46,13 @@ func EmptyContext(logger mlog.LoggerIFace) *Context {
 	}
 }
 
+// TestContext creates an empty context with a new logger to use in testing where a test helper is
+// not required.
+func TestContext() *Context {
+	logger, _ := mlog.NewLogger()
```

Comment:
`mlog.CreateConsoleTestLogger` is a better choice I think

Reply:
Isn't `mlog.CreateTestLogger` better? Or do we want logging to stdout everything on tests?

Reply:
0/5 from my side. Whatever you think is the bets.

Reply:
`CreateTestLogger` is not being used, so I'll stick with `CreateConsoleTestLogger` as suggested.